calcom/cal.com
# fix: use destination calendar email

Fixes: https://github.com/calcom/cal.com/issues/3643
Fixes: https://github.com/calcom/cal.com/issues/5774

How to test?
1) Connect calendar integration (like google) with email address different than the account
2) Go to Event Type Advanced Settings and turn on the switch and click 'Save'

<img width="950" alt="Screenshot 2024-02-08 at 3 13 00 PM" src="https://github.com/calcom/cal.com/assets/53316345/53b97347-98d5-4d23-8d0c-98f3066b275b">

3) Now Book a meeting on the preview URL

Check if email and calendar event is sent to calendar integration email and not account settings email.

Try different combinations:- outlook integration + team event type.. etc

Preference order:- 
Event type Destination email >  Account Email address


TODO

- [x] Updating all the tests
- [x] Testing all edge cases (Individual event type, team event types)



Could break core features
